api/src/permissions/modules/process-payload/process-payload.ts

```diff
@@ -112,13 +118,43 @@ export async function processPayload(options: ProcessPayloadOptions, context: Co
 	if (validationRules.length > 0) {
 		const validationErrors: InstanceType<typeof FailedValidationError>[] = [];
 
-		validationErrors.push(
-			...validatePayload({ _and: validationRules }, payloadWithPresets)
-				.map((error) =>
-					error.details.map((details) => new FailedValidationError(joiValidationErrorItemToErrorExtensions(details))),
-				)
-				.flat(),
-		);
+		if (options.keys && options.keys.length > 0) {
+			const itemService = new ItemsService(options.collection, {
+				accountability: null,
```

Comment:
Should be OK to query as an admin here as it is only merged for the `validatePayload` call.